transports/azure-storage-queues/multi-storageaccount-support.md

```diff
@@ -15,14 +15,13 @@ related:
  - transports/azure-storage-queues/configuration
 ---
 
-Endpoints running on Azure Storage Queues transport using a single storage account is subject to potential throttling once the maximum number of messages is written to the storage account. To overcome this limitation, use multiple storage accounts. To better understand scale out options with storage accounts, it is necessary understand Azure storage account scalability and performance.
-
+Endpoints running on the Azure Storage Queues transport using a single storage account are subject to potential throttling once the maximum number of concurrent requests to the storage account is hit. To overcome this limitation, use multiple storage accounts. To better understand scale out options with storage accounts, it is necessary to understand [Azure storage account scalability and performance](https://docs.microsoft.com/en-us/azure/storage/common/storage-scalability-targets).
 
 ## Azure Storage Scalability and Performance
 
-All messages in a queue are accessed via a single queue partition. A single queue is targeted to process up to 2,000 messages per second. Scalability targets for storage accounts can vary based on region with up to 20,000 messages per second (throughput achieved using an object size of 1KB). This is subject to change and should be periodically verified using [Azure Storage Scalability and Performance Targets](https://docs.microsoft.com/en-us/azure/storage/storage-scalability-targets).
+All messages in a queue are accessed via a single queue partition. A single queue is targeted to process up to 2,000 messages per second. Scalability targets for storage accounts can vary based on region with up to 20,000 messages per second (throughput achieved using an object size of 1KB). This is subject to change and should be periodically verified.
```

Comment:
Is it 2000 msg/s or 2000 iop/s? I thought it is the later. And we require 2 IO per message --> 1000 msgs/s

Reply:
According to [doco](https://docs.microsoft.com/en-us/azure/storage/common/storage-scalability-targets#scalability-targets-for-blobs-queues-tables-and-files)

> Target throughput for single queue (1 KB messages)  Up to 2000 messages per second

Reply:
I also think their doco is wrong though and it's iops